### ISSUE TITLE
Weight decay is too small for Lion

### ISSUE THREAD

**Issue by xiangning-chen** [sinpcw/showcase-optimizer#2]
Hi,

Thanks for testing the Lion optimizer!

I noticed that you used a 10x smaller learning rate for Lion compared to AdamW, which is great.
However, the weight decay for both optimizers are 1e-4, leading to a 10x smaller effective weight decay (LR*WD) for Lion.
Suggested by the paper (Section 5), "Based on our experience, a suitable learning rate for Lion is typically 3-10x smaller than that for AdamW. Since the effective weight decay is LR * WD, the value of weight decay used for Lion is 3-10x larger than that for AdamW in order to maintain a similar strength." 

Could you please use a 10x larger weight decay for Lion. For instance, on ViT-B/16, LR=1e-4, WD=1e-4 for AdamW, and LR=1e-5, WD=1e-3 for Lion. Thank you!

**Comment by sinpcw** [comment]
Thank you for your suggestion. I will experiment again with weight_decay of Lion multiplied by x10.
